tools/SourceKit/tools/sourcekitd/lib/API/CMakeLists.txt

```diff
@@ -16,5 +16,8 @@ endif()
 
 add_sourcekit_library(sourcekitdAPI
   ${sourcekitdAPI_sources}
-  DEPENDS SourceKitSupport
+  DEPENDS
+      SourceKitSupport SourceKitSwiftLang
+      # Clang dependencies, required by SourceKitSwiftLang
+      clangFormat clangToolingCore clangDriver
```

Comment:
I wonder if there's a way with CMake to link `SourceKitSwiftLang` such that its own dependencies don't need to be listed again here? 🤔